tests/test_beatles.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+
+import os
+import json
+
+import pytest
+
+from mirdata import beatles, utils
+
+
+def test_download_success(tmpdir, mocker):
+    data_home = str(tmpdir)
+    mock_validated = mocker.patch.object(utils, 'check_validated')
+    mock_validated.return_value = True
+
+    beatles.download(data_home)
+
+    mock_validated.assert_called_once()
+    assert len(os.listdir(data_home)) == 0
+
+
+def test_download_missing_files(tmpdir, mocker):
+    data_home = str(tmpdir)
+    save_path = utils.get_save_path(data_home)
+    beatles_annotations_path = os.path.join(save_path, beatles.BEATLES_DIR)
+
+    mock_validated = mocker.patch.object(utils, 'check_validated')
+    mock_validated.return_value = False
```

Comment:
If you are using the same mock across multiple tests, extract them to a 

```
@pytest.fixture
def mock_validated(mocker):
  return mocker.patch.object(utils, 'check_validated')

...

def test_download_missing_files(tmpdir, mocker, mock_validated):
  # etc
```

It's a nice way to, using injection only, guarantee that something doesn't get invoked but rather gets mocked.

Reply:
Ooh I was looking for a mechanism to do this. 👍